client/src/app/videos/+video-edit/video-add.component.ts

```diff
@@ -27,6 +27,7 @@ export class VideoAddComponent extends FormReactive implements OnInit {
 
   isUploadingVideo = false
   videoUploaded = false
+  videoUpload = null
```

Comment:
From a quick skim, isn't `videoUpload` redundant with `isUploadingVideo`, and thus we could remove the latter by testing `!!videoUpload`

Reply:
As I do not know the code base very well yet, I didn't want to introduce unforeseen side-effect. Also, isVideoUploading is a boolean flag while videoUpload (very bad name but I was out of idea) is the observable returned by the subscription to the HttpClient request and I was not feeling comfortable testing on it (the upload is finished but for some reason, something did not go entirely as planned for example).
I'm not against it though. 
What do maintainers thinks about that ?

Reply:
Just rename it to `videoUploadObservable` and it should be okay :)